### ISSUE TITLE
how can i add original batch norm layer correctly?

### ISSUE THREAD

**Issue by yzp12** [yjxiong/caffe#118]
Hi,
I want to use the original batch norm layer with your version of caffe. I added the .cu file, .cpp file, and i added the definition of batchnorm layer in commom_layer.hpp. I also modified the caffe.prototxt to add the batch norm layer's message. But when i make, it still turns out a lot of errors like this:
/work/01932/dineshj/CS381V/caffe_install_scripts/install/include/boost/python/object_core.hpp(499): warning: call through incomplete class "boost::python::api::proxy<boost::python::api::const_attribute_policies>" will always produce an error when instantiated

Anyone can shed some light on how could i resolve this? I appreciate a lot!

**Comment by yjxiong** [comment]
This seems to be a warning. Are you able to run your program?

**Comment by yzp12** [comment]
Hi, @yjxiong. My bad. I am able to run it. Thanks a lot!
